README.md

```diff
@@ -84,15 +89,13 @@ public function index()
 The `search` finder and the `filterParams()` method are dynamically provided by the
 `Search` behavior.
 
-* Then add the component search in the necessary methods (for our example index)
-```php
-public function index()
-{
-    $this->loadComponent('Search.Prg');
-}
-```
+### Component
+Then add the Search Prg component to the necessary methods in your controller. 
+
+:warning: Make sure, 
+* That you add this in the controllers `initialize` method. 
```

Comment:
Because the method belongs to the controller? I should probably learn this :cake: